app/classifier/classification-summary.jsx

```diff
@@ -44,8 +47,29 @@ class ClassificationSummary extends React.Component {
     );
   }
 
+  isFeedbackEnabled() {
+    if (this.props.project && this.props.project.experimental_tools.includes('general feedback')) {
```

Comment:
This line is quite difficult to read. Deconstruct to improve that: perhaps `const { project } = this.props` would help.